floodviz/views.py

```diff
@@ -1,7 +1,112 @@
 from flask import request, render_template, abort
+import csv
+import requests
+import yaml
+import os
 
 from . import app
 
+
 @app.route('/home/')
 def home():
-    return render_template('index.html')
\ No newline at end of file
+    return render_template('index.html')
+
+
+# This program requires svgis to be installed (which in turn requires Fiona and GDAL)
+# Takes a list of site ids in the form of "site_ids.csv" (IDs now given in config file)
+# Takes config data in the form of "site_configs.yaml" (also now given in config file)
+# Takes style in the form of "site_styles.css"
+# Adds a city layer in the form of "cities.json"
+# outputs an SVG with ids for each site
+
+siteList = app.config['SITE_IDS']
+
+bounds = app.config['BOUNDS']
+
+# put each site number from the csv into a list
+# with open('static/data/site_ids.csv', newline='') as csvfile:
+#     siteReader = csv.reader(csvfile)
+#     for site in siteReader:
+#         if site[0].isalnum():
+#             siteList.append(site[0])
+
+
+# generate the string of site ids for the url
+idInputString = ",".join(siteList)
+# print(idInputString)
+
+url = "https://waterservices.usgs.gov/nwis/site/?format=rdb&sites=" + idInputString + "&siteStatus=all"
+
+# get data from url
+req = requests.get(url)
+
+# Discard preamble from response
```

Comment:
I suspect this will not work in some cases. If any of the data in the columns contains a '#' sign, you will have removed too much. It may be better to iterate over the lines until you get to the first one that does not start with #.

I would also put this into a separate function within a utils module. Seems like something we might want to reuse and it would be good to write unit tests for it.